.github/CONTRIBUTING.md

```diff
@@ -50,192 +48,260 @@ npm run bootstrap
-      -  Firefox
-      -  IE11 (except speech features)
-   -  Safari on macOS
-   -  Safari on iOS or iPadOS
-   -  Chrome on Android
--  For accessibility, please refer to [`docs/ACCESSIBILITY.md`](https://github.com/microsoft/BotFramework-WebChat/blob/main/docs/ACCESSIBILITY.md)
-   -  Tab order, content readability, assistive technology-only text, color contrast, etc. must be maintained.
-   -  Assistive technology and browser compatibility
-      -  NVDA/JAWS: Chrome and Firefox
-      -  Narrator: Microsoft Edge family and IE11
-      -  VoiceOver: Safari
-      -  TalkBack: Chrome on Android
--  For internationalization, please refer to [`docs/LOCALIZATION.md`](https://github.com/microsoft/BotFramework-WebChat/tree/main/docs/LOCALIZATION.md)
--  Feature documentation, samples, live demo, operation of demo bots
-   -  All samples must also come with a hosted live demo
-   -  Please discuss with us if a specific bot is needed for the live demo
-
-# Workflow
-
-Here list how we generally work when [fixing a bug](#fixing-bug) or [implementing a new feature](#implement-new-feature).
-
-## Fixing bug
-
-Write the bug repro as a test, before fixing the bug.
-
-1. Clone/prepare the repository or reset an existing repository
-   -  To reset the repo, run `npm run tableflip`
-1. Run continuous build, run `npm start`
-1. Convert bug into a test case, under `__tests__/html/this-is-the-bug.html`
-1. Run the test, `npm run start:docker`, then `npm test -- --testPathPattern html/this-is-the-bug.html`
-   -  Make sure the test fail as described in the repro
+- Transparency
+  - Summarize test updates or changes as a part of the pull request
+  - If there are ONLY test changes, summarize those changes in `CHANGELOG.md` as well
+  - Fill out the pull request form with details
+- Hygiene
+  - Make sure imports, members, variables, etc, are sorted alphabetically
+  - Avoid one-off variables, prefer JavaScript shorthands, shorter and faster code
+  - CSS
+    - Remove unneeded CSS styles
+    - Use CSS BEM and always namespace with `webchat__`, for example, `webchat__block__element--modifier`
+    - [Articles on CSS BEM](#articles-related-to-css-block-element-modifier-methodology)
+  - Only use local assets
+    - All assets must be self-contained and not loaded from any external URLs. This includes both Web Chat assets and test assets
+  - No logging to console. Only exceptions:
+    - Deprecation notes
+    - Warnings
+    - Errors
+  - No global pollution, for example:
+    - No `taborder` with numbers other than `0` or `-1`
+    - Minimize `z-index` usage
+      - If `z-index` is an absolute must, it must be used in a [new stacking context](https://developer.mozilla.org/en-US/docs/Web/CSS/CSS_Positioning/Understanding_z_index/The_stacking_context)
+    - Be mindful when using CSS styles in a component with content from end-developers. CSS styles may leak into the content, for example:
+      - Set `font-family` as early as possible
```

Comment:
I added a little more to add here, but it's in a different comment. Could you verify and add?